init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/service/InitProgressReporter.java

```diff
@@ -0,0 +1,41 @@
+package uk.ac.ox.ndph.mts.init_service.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+@Service
+public class InitProgressReporter implements AutoCloseable {
+    private BufferedWriter writer;
+    private static final Logger LOGGER = LoggerFactory.getLogger(InitProgressReporter.class);
+
+    @Autowired
+    public InitProgressReporter(@Value("${INIT_PROGRESS_LOG_PATH:}") String initLogMountPath) throws IOException {
```

Comment:
1. INIT_PROGRESS_LOG_PATH isn't consistent with spring, I guess it should be something like progress.log.path
2. why do you have a ":" at the end?

Reply:
1. I'll change it, correct.
2. don't need it anymore cause I've set an empty string in the docker-compose for the API Tests